packages/container/src/pages/home/Home.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
If you want to have a `/src/pages/home` folder, then this file should be named `/src/pages/home/index.tsx`